kustomize/commands/localize/localize_test.go

```diff
@@ -66,6 +82,58 @@ func TestScopeFlag(t *testing.T) {
 	loctest.CheckFs(t, testDir.String(), expected, actual)
 }
 
+func TestNoVerifyFlag(t *testing.T) {
+	kustomization := map[string]string{
+		"kustomization.yaml": `namePrefix: test-
```

Comment:
I think it would be interesting to have an extra test case here with some localizable content. Perhaps a resource such as https://raw.githubusercontent.com/kubernetes-sigs/kustomize/v1.0.6/examples/helloWorld/deployment.yaml could be a good candidate (since it is part of this repo)?

Reply:
I can also use the deployment variable that has something similar in this test file. I'll change that!